collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -479,6 +501,8 @@ struct cgroup {
     RRDSET *st_queued_ops;
     RRDSET *st_merged_ops;
 
+    // pressure charts are inside `struct pressure' due to complexity
```

Comment:
Do you really need this comment? Moreover, it's in the wrong place.

Reply:
I put it there because I see RRD*s for all other charts are put directly into `struct cgroup`. It is mostly a note in case people may not know where to look for the pressure RRD*s and get confused. I'm surely willing to remove it if it's useless.

Reply:
I think you can remove it.